contracts/cw721-base/src/multi_tests.rs

```diff
@@ -133,3 +133,36 @@ fn test_016_017_migration() {
         .unwrap();
     assert_eq!(m.minter, admin().to_string());
 }
+
+/// Test backward compatibility using instantiate msg from a 0.16 version on latest contract.
+/// This ensures existing 3rd party contracts doesnt need to updated as well.
+#[test]
+fn test_instantiate_016_msg() {
+    use cw721_base_016 as v16;
+    let mut app = App::default();
+    let admin = || Addr::unchecked("admin");
+
+    let code_id_latest = app.store_code(cw721_base_latest_contract());
+
+    let cw721 = app
+        .instantiate_contract(
+            code_id_latest,
+            admin(),
+            &v16::InstantiateMsg {
+                name: "collection".to_string(),
+                symbol: "symbol".to_string(),
+                minter: admin().into_string(),
+            },
+            &[],
+            "cw721-base",
+            Some(admin().into_string()),
+        )
+        .unwrap();
+
+    // assert withdraw address is None
+    let withdraw_addr: Option<String> = app
+        .wrap()
+        .query_wasm_smart(cw721, &crate::QueryMsg::<Empty>::GetWithdrawAddress {})
+        .unwrap();
+    assert!(withdraw_addr.is_none());
+}
```

Comment:
Test backward compatibility using instantiate msg from a 0.16 version on latest contract. This ensures existing 3rd party contracts doesnt need to be updated.
Basically this PR adds an optional feature, existing features behaves as-is.